uberspace/deploy.yml

```diff
@@ -99,6 +99,29 @@
           path: ~/uwsgi/out.log
           state: touch
 
+    - name: install and set up redis cache
+      block:
+      - name: create redis directory
+        file:
+          path: ~/.redis/
+          state: directory
+      - name: copy redis conf
+        template:
+          src: "files/redis.conf.j2"
+          dest: "~/.redis/conf"
+      - name: copy redis ini
+        template:
+          src: "files/redis.ini.j2"
+          dest: "~/etc/services.d/redis.ini"
+      - name: reread supervisorctl
+        supervisorctl:
+          name: redis
+          state: present
+      - name: restart redis
+        supervisorctl:
+          name: redis
+          state: restarted
```

Comment:
Can you implement the rereading (and the restart) as a handler? It takes a while and it only needs to be reread if there's a change. Also, there's currently no reread for when the uwsgi config changes, is there? Maybe that affects the emails as well?

Reply:
i moved that hint to the issue